nr_logger.py

```diff
@@ -0,0 +1,102 @@
+import logging
+import time
+
+from process import Process
+
+
+class LogWriter(Process):
+  """ Reads data from the logging queue and writes it to a file. """
+
+  def __init__(self, messenger, filename):
+    """ Args:
+      messenger: The Messanger instance to use.
+      filename: The base file name to write log messages to. """
+    super(LogWriter, self).__init__(messenger)
+
+    # Use the current system time to make each new file unique.
+    systime = int(time.time())
+    self.__file = open("%s-%d.log" % (filename, systime), "a")
+
+  def __del__(self):
+    self.__file.close()
+
+  def _run(self):
+    log_queue = self._messenger.get_queue("logging")
+
+    # Keep reading messages and write them to the file.
+    while True:
+      try:
+        message = log_queue.get()
```

Comment:
Should standardize message payload to include process name and class name as well as date/time.